nx584/config.yaml

```diff
@@ -11,3 +14,9 @@ arch:
   - i386
 ports:
   5007/tcp: 5007
+options:
+  serial_dev: "/dev/serial/by-id/usb-Prolific_Technology_Inc._USB-Serial_Controller-if00-port0"
```

Comment:
This is hard-coded to your own serial device. Ideally it should be something more generic?

Reply:
yes, that's my config, the idea is that this parameter can be easily set in addon config tab